src/main/resources/view/PatientListCard.fxml

```diff
@@ -13,9 +13,9 @@
 
   <GridPane HBox.hgrow="ALWAYS">
     <columnConstraints>
-      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="0" prefWidth="150" />
     </columnConstraints>
-    <VBox alignment="CENTER_LEFT" minHeight="105" GridPane.columnIndex="0">
+    <VBox alignment="CENTER_LEFT" minHeight="85" GridPane.columnIndex="0">
```

Comment:
Nice! We need to also ensure that the GUI is still responsive after having side panes.